src/qililab/experiment/base_experiment.py

```diff
@@ -127,7 +127,7 @@ def _threaded_function():
                 if self.results_path is not None:
                     with open(file=self.results_path / "results.yml", mode="a", encoding="utf8") as data_file:
                         result_dict = result.to_dict()
-                        yaml.safe_dump(data=[result_dict], stream=data_file, sort_keys=False)
+                        YAML().dump(data=[result_dict], stream=data_file)
```

Comment:
Can we use `sort_keys=False` with ruamel to make sure the functionality is the same?

Reply:
In the same post, you can see that there is actually a `.sort()` method, but for the class `YAML()`, so you first would need to have the yaml in such class, order it as you want with lambas, and then dump it.

But as said before, I think we don't need this, since the Default way already conserves the order!

The only problem we would have would be, in the opposite ways, when we actually want a alphabetical order, but I don't think we want that anywhere, no?

Reply:
Did this in my terminal to check it:
![Uploading Screenshot 2023-10-25 at 19.26.26.png…]()

Reply:
We don't want to sort anything, thus we are good! Good job! 🚀 💪